src/analysis/souffle/sensitivity_analysis_unit_test_interface.dl

```diff
@@ -0,0 +1,16 @@
+#ifndef SENSITIVITY_ANALYSIS_UNIT_TEST_INTERFACE_DL_
+#define SENSITIVITY_ANALYSIS_UNIT_TEST_INTERFACE_DL_
+
+#include "src/analysis/souffle/sensitivity_analysis.dl"
+
+//An interface used for unit testing of the sensitivity analysis. 
+//Only outputs value sensitivities
+
+.input isOperation(delimiter=";")
+.input isInitialInput(delimiter=";")
```

Comment:
I'm intrigued by the decision to make `isInitialInput` an input relation. Wouldn't it be safer to just look for operations that grab columns out of tables for which there are no predecessors to discover the set of initial inputs?

Another thought: we basically only care about sensitivity analysis for columns that have user data or other sensitive information in them. So perhaps this should instead be something like `isSensitiveColumn`.

Reply:
Is this a concern so that the user doesn't have to add any more annotations? This relation is supposed to distinguish data sources that have been given as input vs. intermediate or final ones produced by running queries on them, so not entirely sure isSensitiveColumn makes that difference clear.

Reply:
Yes, I introduced this change in the other PR which depends on this.